L/libnovas/build_tarballs.jl

```diff
@@ -0,0 +1,54 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "libnovas"
+version = v"3.1.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/kiranshila/novas.git", "0d4d08650d3b8a0f798da2b86f894b5083654e32")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/novas
+autoreconf -i -f
```

Comment:
Meson may be a more user-friendly build system than Autoconf, if you need to roll your own.  You can see a simple example in https://github.com/JuliaPackaging/BinaryBuilder.jl/blob/ebeec31948c7cd584be5cfe29ea8a7e47db213b8/test/build_tests/libfoo/meson.build